src/main/webapp/src/app/recommendation-form/recommendation-form.component.html

```diff
@@ -0,0 +1,74 @@
+<h1>Hidden Gems Recommendations</h1>
+<h4>Get a restaurant recommendation tailored just for you!</h4>
+
+<form [formGroup]="preferenceForm" (ngSubmit)="onSubmit()">
+  <h2>Your Restaurant Preferences</h2>
+  <p>Please select your restaurant preferences in the form below. If you have no preference for a field, please feel free to leave it blank</p>
+  
+  <h3>
+    <label for="cuisine">
+        Cuisine
+        <br>
+        <br>
+        <input type="text" formControlName="cuisine" value="cuisine">
+      </label>
+  </h3>
+
+  <h3>Price Range</h3>
+  <div  class="price-group">
+    <label for="low">
+      💲
+      <input type="radio" formControlName="price" value="low">
+    </label>
+    <label for="medium">
+      💲💲
+      <input type="radio" formControlName="price" value="medium" >
+    </label>
+    <label for="high">
+      💲💲💲
+      <input type="radio" formControlName="price" value="high" >
+    </label>
+    <label for="extreme">
+      💲💲💲💲
+      <input type="radio" formControlName="price"value="extreme" >
+    </label>
+  </div>
+
+  <h3>Minimum star rating preferred</h3> 
+  <div class="rating-group">
+    <label for="1">
+      ⭐️
+      <input type="radio" formControlName="rating" value="1">
+    </label>
+    <label for="2">
+      ⭐️ ⭐️
+      <input type="radio" formControlName="rating" value="2">
+    </label>
+    <label for="3">
+      ⭐️ ⭐️ ⭐️
+      <input type="radio" formControlName="rating" value="3">
+    </label>
+    <label for="4">
+      ⭐️ ⭐️ ⭐️ ⭐️
+      <input type="radio" formControlName="rating" value="4">
+    </label>
+    <label for="5">
+      ⭐️ ⭐️ ⭐️ ⭐️ ⭐️
+      <input type="radio" formControlName="rating" value="5">
+    </label>
+  </div>
+
+  <h3>
+    <label>Restaurant Type</label>
+  </h3>
+  <select formControlName="type">
```

Comment:
Similarly as the comment above, from what I understood, we can only check 'restaurant' or 'cafe' for now. I think it's fine to keep all, but maybe later you can add a comment or something to indicate the 2 other options won't be functional unless we use another API, or something like that.

Reply:
fixed